ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -53,6 +53,21 @@ class DemoListViewController: UITableViewController {
         tableView.register(TableViewCell.self, forCellReuseIdentifier: cellReuseIdentifier)
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if didAutoLaunch {
+            UserDefaults.standard.set(0, forKey: DemoListViewController.lastDemoControllerKey)
+        } else {
+            let lastDemoController = UserDefaults.standard.integer(forKey: DemoListViewController.lastDemoControllerKey)
+            if lastDemoController != 0 {
+                tableView(tableView, didSelectRowAt: IndexPath(row: lastDemoController - 1, section: 0))
```

Comment:
I'm thinking about the out of bounds scenario here and/or when demos list changes.

I think storing the title in the demos list could be a better approach because:
 1. you can check whether the item is present on the list (in case the app updates and the item was renamed or removed)
 2. you can remove the UserDefaults value and not load any other demo view controller in case it was not found
 3. If new items are added to the list (before the current default index), finding it by the title won't cause a different one to be presented.

Then, instead of calling func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath), you can just use its logic once you have the title of the demo controller you want.

Reply:
Updated to using the demo controller name instead.